### DISCUSSION TITLE
Remove console logging of environment variables

### DISCUSSION THREAD

**Discussion by orenelbaum** [solidjs/solid-start#569] [Ideas]
Environment variables are always being logged to the console in dev, and it creates unnecessary noise in the console.
I think that those logs should probably be removed.

**Comment by ryansolid** [comment]
Hmm.. I thought the complaint were these being in prod. But for dev given all the modes SolidStart has they are kinda helpful. I just used them the other day debugging. I believe these are only Client available values. I'm going to open this up to discussion.
